Modules/_decimal/_decimal.c

```diff
@@ -3194,16 +3239,24 @@ dec_new_impl(PyTypeObject *type, PyObject *value, PyObject *context)
     return PyDecType_FromObjectExact(type, value, context);
 }
 
-static PyObject *
-ctx_create_decimal(PyObject *context, PyObject *args)
-{
-    PyObject *v = NULL;
+/*[clinic input]
+_decimal.Context.create_decimal
 
-    if (!PyArg_ParseTuple(args, "|O", &v)) {
-        return NULL;
-    }
+    self as context: self
+    num: object(c_default="NULL") = "0"
```

Comment:
It is easy to rename variables.
```suggestion
    number: object(c_default="NULL") = "0"
```

Reply:
The cost is yet another incompatibility with pure-Python version do we want this?